fake_switches/cisco/command_processor/enabled.py

```diff
@@ -72,11 +72,18 @@ def do_show(self, *args):
             for vlan in sorted(self.switch_configuration.vlans, key=lambda v: v.number):
                 ports = [port.get_subname(length=2) for port in self.switch_configuration.get_physical_ports()
                          if port.access_vlan == vlan.number or (vlan.number == 1 and port.access_vlan is None)]
+                formatted_membership = []
+                if ports:
+                    ports_membership = ["    {}".format(l) for l in get_port_groups(ports, max_line_length=30)]
+                    formatted_membership.append(ports_membership.pop(0))
+                    for remaining_line in ports_membership:
+                        formatted_membership.append('                                            {}'.format(remaining_line))
```

Comment:
ah just got what you're doing here...
maybe you could use a format similar to line 82 where a number shows the white spaces?